oap-server/server-cluster-plugin/pom.xml

```diff
@@ -47,5 +47,10 @@
             <artifactId>library-util</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.skywalking</groupId>
+            <artifactId>library-client</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
You should remove this dependency totally.

Reply:
done